locales/nb_NO/pcsx2_Main.po

```diff
@@ -68,19 +69,18 @@ msgstr "Filen ble ikke funnet."
 
 #: common/src/Utilities/Exceptions.cpp:354
 msgid ""
-"Permission denied while trying to open file, likely due to insufficient user "
-"account rights."
+"Permission denied while trying to open file, likely due to insufficient user account "
```

Comment:
This one too.

Reply:
Again... I do not remember doing those changes. I only changed the Norwegian strings.